examples/03_connectivity/plot_group_level_connectivity.py

```diff
@@ -154,84 +136,81 @@ def plot_matrices(matrices, matrix_kind):
-kinds = ['correlation', 'partial correlation', 'tangent']
-for kind in kinds:
-    conn_measure = ConnectivityMeasure(kind=kind, vectorize=True)
-    connectivity_biomarkers[kind] = conn_measure.fit_transform(pooled_subjects)
-
-# For each kind, all individual coefficients are stacked in a unique 2D matrix.
-print('{0} correlation biomarkers for each subject.'.format(
-    connectivity_biomarkers['correlation'].shape[1]))
-
-###############################################################################
-# Note that we use the **pooled groups**. This is crucial for **tangent** kind,
-# to get the displacements from a **unique** `mean_` of all subjects.
-
-###############################################################################
-# We stratify the dataset into homogeneous classes according to phenotypic
-# and scan site. We then split the subjects into 30 folds with the same
-# proportion of each class as in the whole cohort for cross-validation
+# we will use connectivity matrices as features to distinguish children from
+# adults. We use cross-validation and measure classification accuracy to
+# compare the different kinds of connectivity matrices.
+# We use random splits of the subjects into training/testing sets.
+# StratifiedShuffleSplit allows preserving the proportion of children in the
+# test set.
+from sklearn.svm import LinearSVC
 from sklearn.model_selection import StratifiedShuffleSplit
+from sklearn.metrics import accuracy_score
+import numpy as np
 
+kinds = ['correlation', 'partial correlation', 'tangent']
 _, classes = np.unique(groups, return_inverse=True)
-cv = StratifiedShuffleSplit(n_splits=30)
-###############################################################################
-# and use the connectivity coefficients to classify children vs adults.
+cv = StratifiedShuffleSplit(n_splits=15, random_state=0, test_size=5)
+pooled_subjects = np.asarray(pooled_subjects)
 
-# Note that in cv.split(X, y),
-# providing y is sufficient to generate the splits and
-# hence np.zeros(n_samples) may be used as a placeholder for X
-# instead of actual training data.
-from sklearn.svm import LinearSVC
-from sklearn.model_selection import cross_val_score
-
-mean_scores = []
+scores = {}
 for kind in kinds:
-    svc = LinearSVC(random_state=0)
-    cv_scores = cross_val_score(svc,
-                                connectivity_biomarkers[kind],
-                                y=classes,
-                                cv=cv,
-                                groups=groups,
-                                scoring='accuracy',
-                                )
-    mean_scores.append(cv_scores.mean())
-
-###############################################################################
-# Finally, we can display the classification scores.
-
-###############################################################################
-# Finally, we can display the classification scores.
-from nilearn.plotting import show
+    print(kind)
```

Comment:
I'm not sure what this print statement adds, other than having an output between the cells. Slight stylistic preference to drop -- feel free to ignore.